jeromeetienne/AR.js
# Y position for gps entity place

**⚠️ All PRs have to be done versus 'dev' branch, so be aware of that, or we'll close your issue ⚠️**

**What kind of change does this PR introduce?**
A new feature.

**Can it be referenced to an Issue? If so what is the issue # ?**
#600 

**How can we test it?**


Just add an yPosition parameter to some gps-entity-place and check that the position on y axis changed.

**Summary**

PR adds a parameter yPosition to gps-entity-place and allows to change positions of object on y-axis while in location mode.

**Does this PR introduce a breaking change?**
The newly introduced parameter is optional and if you don't specify it gps-entity-place will behave in the same way as before change. 

**Please TEST your PR before proposing it. Specify here what device you have used for tests, version of OS and version of Browser**

Smartphone: Xiaomi Mi A1, Android 9, Chrome v79.0.3945.116

**Other information**
